core/core.py

```diff
@@ -557,14 +556,275 @@ def show_options(self, options=None):
 		else:
 			print(f'{os.linesep}{self.spacer}No options available for this module.{os.linesep}')
 
-	def show_var(self):
-		self.do_var('list')
-
 	def _get_show_names(self):
 		prefix = 'show_'
 		return [x[len(prefix):]
 				for x in self.get_names() if x.startswith(prefix)]
 
+	# ////////////////////////////////
+	#         INSTALL and DEV       //
+	# ////////////////////////////////
+
+	def do_package(self, params):
+		'''Install extensions and packages and manage them.'''
+		params = params.split()
+		if not params or len(params) < 3:
+			self.help_package()
+			return
+		mode = params.pop(0).lower()
+		if mode == 'extension':
+			next_plan = params.pop(0).lower()
+			pool = self.request(
+				'https://raw.githubusercontent.com/mexts/init/main/EXTENSIONS.json').json()
+			if next_plan == 'list':
+				self.alert('List of extensions:')
+				self.alert_results(pool)
+			elif next_plan == 'install':
+				for name in params:
+					name = params.pop(0).lower()
+					if name not in pool:
+						self.error(f"There's no '{name}' extension name.", 'core', 'do_install')
+						continue
+					exts = 'https://raw.githubusercontent.com/mexts/init/main/exts/'
+					mext = self.request(f"{exts}/{name}/mext").text
+					mext = mext.split('\n')
+					if '' in mext:
+						mext.pop(mext.index(''))
+					try:
+						mext[-1] = json.loads(mext[-1])
+					except Exception as e:
+						self.error(f"mext file is missed.", 'core', 'do_package')
+						self.print_exception()
+						return
```

Comment:
If we replaced this with `continue` wouldn't it try to install all packages and not stop when there is an error?